components/modelmeshserving/modelmeshserving.go

```diff
@@ -22,6 +22,7 @@ import (
 var (
 	ComponentName          = "model-mesh"
 	Path                   = deploy.DefaultManifestPath + "/" + ComponentName + "/overlays/odh"
+	monitoringPath         = deploy.DefaultManifestPath + "/" + "modelmesh-monitoring/base"
```

Comment:
@zdtsw Do we need to pull the modelmesh-monitoring  manifests in Dockerfile as well?

Reply:
it is still here https://github.com/red-hat-data-services/rhods-operator/blob/rhods-2.5/get_all_manifests.sh#L57 but will need handle in midstream